### PR TITLE
Link the Cirrus CI configuration builder

### PR DESCRIPTION
**Pull request by RDIL**
This can help people who are a bit new to the scene by allowing them to use a GUI to configure their tasks.

### CODE REVIEW

File: docs/guide/writing-tasks.md [hunk 1]
```diff
@@ -53,6 +53,10 @@ task:
 
     **Note:** instructions within a task can only be named via a prefix (e.g. `test_script`).
 
+!!! tip "Task Creation for Beginners"
```
Comment (fkorotkov) [inline docs/guide/writing-tasks.md:56]:
```suggestion
!!! tip "Visual Task Creation for Beginners"
```

File: docs/guide/writing-tasks.md [hunk 2]
```diff
@@ -53,6 +53,10 @@ task:
...
+    If you are just getting started and prefer a more simple way of creating tasks, the
```
Comment (fkorotkov) [inline docs/guide/writing-tasks.md:57]: "simple" here is questionable. IMO copy and pasting from examples is the simplest way.

```suggestion
    If you are just getting started and prefer a more visual way of creating tasks, there
```

File: docs/guide/writing-tasks.md [hunk 3]
```diff
@@ -53,6 +53,10 @@ task:
...
+    [Cirrus CI Configuration Builder](https://cirrusbuilder.rdil.rocks) might help.
```
Comment (fkorotkov) [inline docs/guide/writing-tasks.md:58]:
```suggestion
    is a third-party [Cirrus CI Configuration Builder](https://cirrusbuilder.rdil.rocks) for generating YAML config that might be helpful.
```

File: docs/guide/writing-tasks.md (hunk 2 above)
Comment (RDIL) [reply docs/guide/writing-tasks.md:57]: 'the' should not be 'there' in this case.

Comment (fkorotkov) [reply docs/guide/writing-tasks.md:57]: change it to work with the next line "there is a third-party"

Comment (RDIL) [reply docs/guide/writing-tasks.md:57]: Ah

**Pull request merged**